### PR TITLE
added universal app

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -416,6 +416,20 @@
         "url": "https://raw.githubusercontent.com/feedhenry/template-config/master/screenshots/apps/blank_xamarin/1.png"
       }]
     }, {
+      "id": "blank_vs_universal",
+      "name": "Blank Visual Studio C# Universal App",
+      "type": "client_native_windows",
+      "description": "Hello World Xamarin App",
+      "priority": 0.03,
+      "repoUrl": "git://github.com/edewit/fh-csharp-universal-sdk-blank-app.git",
```
Comment (cianclarke) [inline global.json]: Should be in the feedhenry-templates github org

File: global.json [hunk 2]
```diff
@@ -416,6 +416,20 @@
         "url": "https://raw.githubusercontent.com/feedhenry/template-config/master/screenshots/apps/blank_xamarin/1.png"
       }]
     }, {
+      "id": "blank_vs_universal",
+      "name": "Blank Visual Studio C# Universal App",
+      "type": "client_native_windows",
+      "description": "Hello World Xamarin App",
```
Comment (cianclarke) [inline global.json]: Updated description?

**Comment by cianclarke** [comment]
I'd probably have @wei-lee review git://github.com/edewit/fh-csharp-universal-sdk-blank-app.git also?

File: global.json (hunk 1 above)
Comment (edewit) [inline global.json]: added issue https://issues.jboss.org/browse/FH-95 to track this

**Comment by edewit** [comment]
@wei-lee any comments?

**Comment by wei-lee** [comment]
@edewit When I open the sample project in VS express 2013, I can't build the project straight away as the dependencies can not resolved?

Also we should use the release version 1.3.0 of the FH SDK instead.

**Comment by edewit** [comment]
@wei-lee this pr fixes that https://github.com/feedhenry/fh-dotnet-sdk/pull/11

**Comment by edewit** [comment]
@wei-lee can we merge this?

**Comment by wei-lee** [comment]
@edewit I think so. Users will not be able to build this app on the platform, right?

**Comment by edewit** [comment]
@wei-lee right that is why we have this different app type

**Comment by wei-lee** [comment]
@edewit Cool. You can merge this.

**Pull request merged**